app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
```

Comment:
Consider adding a descriptive error message to your `NotWearingMaskError` class. For example, `class NotWearingMaskError(Exception):
    def __init__(self, message="The person is not wearing a mask."):
        super().__init__(message)`